### PR TITLE
zh-translation: docs/tasks/traffic-management/tcp-traffic-shifting/in…

### PR DESCRIPTION
**Pull request by orangegzx**
…dex.md

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 1]
```diff
@@ -9,68 +9,53 @@ owner: istio/wg-networking-maintainers
...
-
-    * 第一步，为测试 TCP 流量转移创建命名空间
-
-        {{< text bash >}}
-        $ kubectl create namespace istio-io-tcp-traffic-shifting
-        {{< /text >}}
+## 设置测试环境{#set-up-the-test-environment}
 
-    * 如果使用[手动注入 sidecar](/zh/docs/setup/additional-setup/sidecar-injection/#manual-sidecar-injection)，请使用下面命令：
+1.  首先，创建一个 Namespace 用于测试 TCP 流量迁移，并将其标记为使用自动注入 Sidecar 方式。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/tcp-traffic-shifting/index.md:24]: namespace 翻译。

File: content/zh/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 2]
```diff
@@ -9,68 +9,53 @@ owner: istio/wg-networking-maintainers
...
-        {{< text bash >}}
-        $ kubectl label namespace istio-io-tcp-traffic-shifting istio-injection=enabled
-        {{< /text >}}
+    {{< text bash >}}
+    $ kubectl create namespace istio-io-tcp-traffic-shifting
+    $ kubectl label namespace istio-io-tcp-traffic-shifting istio-injection=enabled
+    {{< /text >}}
 
-        然后，只需使用 `kubectl` 部署服务即可。
+1. 部署 [Sleep]({{< github_tree >}}/samples/sleep) 应用程序，作为发送请求的测试源。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/tcp-traffic-shifting/index.md:31]: sleep 不用大写吧。

File: content/zh/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 3]
```diff
@@ -9,68 +9,53 @@ owner: istio/wg-networking-maintainers
...
-1. 接下来, 将目标为微服务 `tcp-echo` 的 TCP 流量全部路由到 `v1` 版本。
+1. 部署 `tcp-echo` 微服务的 `v1` 和 `v2` 版本。
 
     {{< text bash >}}
-    $ kubectl apply -f @samples/tcp-echo/tcp-echo-all-v1.yaml@ -n istio-io-tcp-traffic-shifting
+    $ kubectl apply -f @samples/tcp-echo/tcp-echo-services.yaml@ -n istio-io-tcp-traffic-shifting
     {{< /text >}}
 
-1. 确认 `tcp-echo` 服务已启动并开始运行。
+1. 根据[确定 Ingress IP 和 端口](/zh/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports) 中的说明，定义 `TCP_INGRESS_PORT` and `INGRESS_HOST` 环境变量.
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/tcp-traffic-shifting/index.md:43]: 和端口不空格。

File: content/zh/docs/tasks/traffic-management/tcp-traffic-shifting/index.md (hunk 1 above)
Comment (orangegzx) [reply content/zh/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]: 1

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**